src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -2419,6 +2419,13 @@ public ObjectId getHeadRev(String url, String branchSpec) throws GitException, I
     /** {@inheritDoc} */
     public Map<String, ObjectId> getRemoteReferences(String url, String pattern, boolean headsOnly, boolean tagsOnly)
             throws GitException, InterruptedException {
+        return getRemoteReferences(url, pattern != null ? Collections.singletonList(pattern) : Collections.EMPTY_LIST,
+                headsOnly, tagsOnly);
+    }
+
+    /** {@inheritDoc} */
```

Comment:
> leave blank if you have not decided whether the inherited contract has been changed or not yet

imho, in this case better write this explicitly "this method doesn't follow contract, undefined behaviour" rather than implicitly guess that this method is not end in implementation and you can't trust in it's behaviour.